build.rs

```diff
@@ -31,14 +31,10 @@ fn main() {
         compile_windows();
     } else if target.contains("darwin") {
         compile_macos();
-    } else if target.contains("freebsd") {
-        compile_freebsd();
-    } else if target.contains("openbsd") {
-        compile_openbsd();
     } else if target.contains("illumos") {
         compile_illumos();
     } else {
-        panic!("Unsupported target os for hidapi-rs");
+        compile_unix();
```

Comment:
Falling back to generic unix for every unsupported platform is not good enough.
Please make it an appropriate else if .